src/private/files.ts

```diff
@@ -258,4 +336,63 @@ export namespace files {
 
     sendMessageToParent('files.openCloudStorageFile', [file, providerCode, fileOpenPreference]);
   }
+
+  /**
+   * @private
+   * Allow 1st party apps to call this function to get the external
+   * third party cloud storage accounts that the tenant supports
+   * @param excludeAddedProviders: return a list of support third party
+   * cloud storages that hasn't been added yet.
+   */
+  export function getExternalProviders(
+    excludeAddedProviders = false,
+    callback: (error: SdkError, providers: IExternalProvider[]) => void,
+  ): void {
+    ensureInitialized(FrameContexts.content);
+
+    if (!callback) {
+      throw new Error('[files.getExternalProviders] Callback cannot be null');
+    }
+
+    sendMessageToParent('files.getExternalProviders', [excludeAddedProviders], callback);
+  }
+
+  /**
+   * @private
+   * Allow 1st party apps to call this function to move files
+   * among SharePoint and third party cloud storages.
+   */
+  export function copyMoveFiles(
+    selectedFiles: CloudStorageFolderItem[] | ISharePointFile[],
+    providerCode: CloudStorageProvider,
+    destinationFolder: CloudStorageFolderItem | ISharePointFile,
+    destinationProviderCode: CloudStorageProvider,
+    isMove = false,
+    callback: (error?: SdkError) => void,
+  ): void {
+    ensureInitialized(FrameContexts.content);
+    if (isMove === undefined) {
```

Comment:
NIT: This check can never be false because you are providing a default value for `isMove` in the method params.